docs/cli.md

```diff
@@ -63,22 +63,22 @@ that image and add them to the tar streamed to the container into `/artifacts`.
 
 | Name                       | Description                                             |
 |:-------------------------- |:--------------------------------------------------------|
-| `--callbackURL`            | URL to be invoked after successful build (see [Callback URL](#callback-url)) |
+| `--callback-url`            | URL to be invoked after successful build (see [Callback URL](#callback-url)) |
 | `--incremental`            | Try performing an incremental build |
 | `-e (--env)`               | Environment variables passed to the builder eg. `NAME=VALUE,NAME2=VALUE2,...` |
-| `--forcePull`              | Always pull the builder image, even if it is present locally |
+| `--force-pull`              | Always pull the builder image, even if it is present locally |
 | `-l (--location)`          | Location where the scripts and sources will be placed prior doing build (see [STI Scripts](#sti-scripts))|
 | `-r (--ref)`               | A branch/tag from which the build should happen (applies only to GIT source) |
 | `--rm`                     | Remove previous image during incremental build |
-| `--saveTempDir`            | Save the working directory used for fetching scripts and sources |
-| `--contextDir`             | Allow to specify directory name with your application |
+| `--save-temp-dir`            | Save the working directory used for fetching scripts and sources |
+| `--context-dir`             | Allow to specify directory name with your application |
 | `-s (--scripts)`           | URL of STI scripts (see [STI Scripts](#sti-scripts)) |
 | `-q (--quiet)`             | Operate quietly, suppressing all non-error output |
```

Comment:
can you please add --dockercfg-path here (and check if there is something missing ;-) I know, I know, my fault, I should add that ;-)